xbmc/addons/kodi-addon-dev-kit/doxygen/Skin/skin.dox

```diff
@@ -0,0 +1,27 @@
+/*!
+
+\page skin_parts Skin Development
+\brief \htmlonly
+      <h3><span  style="text-decoration: underline;"><span  style="font-style: italic;"><span
+               style="color: rgb(102, 102, 102);">Skin Add-On Development</span></span></span></h3>
+       \endhtmlonly
+
+Kodi includes a GUI library that allows you to skin/change everything you see in
+Kodi, from the images, the sizes and positions of all controls, colours, fonts,
+and text, through to altering navigation and even adding new functionality.
+
+The skin system is quite complex, and this portion of the manual is dedicated to
```

Comment:
Just a suggestion to break up your long sentences: The skin system is quite complex. This portion of the manual is dedicated to providing in depth information on how it all works. It contains tips to make the experience a little more pleasant.